code/modules/mob/new_player/new_player.dm

```diff
@@ -142,9 +142,9 @@
 				to_chat(usr, "<span class='warning'>Spawning currently disabled, please observe.</span>")
 				return
 			var/datum/job/job_datum = locate(href_list["job_selected"])
-			if(!isxenosjob(job_datum) && (SSmonitor.current_state == XENOS_LOSING || SSmonitor.current_state == XENOS_DELAYING))
+			if(!isxenosjob(job_datum))
 				var/datum/job/xeno_job = SSjob.GetJobType(/datum/job/xenomorph)
-				if((xeno_job.total_positions-xeno_job.current_positions) >= 3)
+				if((SSmonitor.gamestate == SHUTTERS_CLOSED || (SSmonitor.gamestate == GROUNDSIDE && SSmonitor.current_state == XENOS_LOSING)) && ((xeno_job.total_positions-xeno_job.current_positions) >= GLOB.alive_xeno_list.len * 0.2))
```

Comment:
Magic number

Reply:
I would also say that one of these checks would be better suited on the lines above to make this one shorter